csharp/language-reference/operators/StackallocOperator.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+
+namespace operators
+{
+    public static class StackallocOperator
+    {
+        public static void Examples()
+        {
+            Console.WriteLine("No snippets with output");
+        }
+
+        private static void AssignToPointer()
+        {
+            // <SnippetAssignToPointer>
+            unsafe
+            {
+                int length = 3;
+                int* numbers = stackalloc int[length];
+                for (var i = 0; i < length; i++)
+                {
+                    numbers[i] = i;
+                }
+            }
+            // </SnippetAssignToPointer>
+        }
+
+        private static void AssignToSpan()
+        {
+            // <SnippetAssignToSpan>
+            int length = 3;
+            Span<int> numbers = stackalloc int[length];
+            for (var i = 0; i < length; i++)
+            {
+                numbers[i] = i;
+            }
+            // </SnippetAssignToSpan>
+        }
+
+        private static void AsExpression()
+        {
+            // <SnippetAsExpression>
+            int length = 1000;
+            Span<byte> buffer = length <= 1024 ? stackalloc byte[length] : new byte[length];
+            buffer = stackalloc byte[128];
```

Comment:
I'm not sure what the intent of this line is.  Should it be removed?

(Note: I'm looking at this before reviewing the associated doc.)

Reply:
@BillWagner intent is to show that one can assign a new stack allocated block to an *earlier* declared variable of `Span` type. The preceding text in the article mentions assignment expression. However, I agree that snippet itself looks weak, not coherent; I was doubting as well if to include that line.

Reply:
@BillWagner how does that line look in the context of the article? If still not fitting, I'll remove it.

Reply:
I would remove this line from the sample:

```csharp
buffer = stackalloc byte[128];
```

The text states that assignment is legal. This extra assignment, without some use of the preceding value doesn't add anything.

Reply:
@BillWagner I updated the snippet.